# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: netwerk/protocol/http/nsHttpChannel.cpp
# Commit: aafdd02eb30a
# Full Hash: aafdd02eb30a2dae68eefb1505083254634d8034
# Author: Paul Bone <pbone@mozilla.com>
# Date: 2019-11-11 21:52:52
# Regressor Bug: 1588412
# File Overlap Count: 2
# Description:
#   Bug 1588412 - Fix a spelling error r=mattwoodrow
#   
#   The already-landed patches contained a spelling error.  This patch fixes it.
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D51390
# ==============================================================================

diff -r aab04214324f -r aafdd02eb30a netwerk/protocol/http/nsHttpChannel.cpp
--- a/netwerk/protocol/http/nsHttpChannel.cpp	Mon Nov 11 08:25:33 2019 +0000
+++ b/netwerk/protocol/http/nsHttpChannel.cpp	Mon Nov 04 21:45:10 2019 +0000
@@ -2581,14 +2581,14 @@
       return NS_OK;
     }
 
-    AssertNotDocumentChennel();
+    AssertNotDocumentChannel();
   }
 
   // No process switch needed, continue as normal.
   return ContinueProcessResponse2(rv);
 }
 
-void nsHttpChannel::AssertNotDocumentChennel() {
+void nsHttpChannel::AssertNotDocumentChannel() {
   if (!mLoadInfo || !IsDocument()) {
     return;
   }
@@ -7773,7 +7773,7 @@
       return NS_OK;
     }
 
-    AssertNotDocumentChennel();
+    AssertNotDocumentChannel();
   }
 
   // No process change is needed, so continue on to ContinueOnStartRequest1.